processing/src/main/java/org/apache/druid/query/aggregation/AggregatorUtil.java

```diff
@@ -121,6 +121,10 @@
   public static final byte MOMENTS_SKETCH_BUILD_CACHE_TYPE_ID = 0x36;
   public static final byte MOMENTS_SKETCH_MERGE_CACHE_TYPE_ID = 0x37;
 
+  // Datasketches Quantiles sketch aggregator (part 2)
+  public static final byte QUANTILES_DOUBLES_SKETCH_TO_RANK_CACHE_TYPE_ID = 0x38;
```

Comment:
Although some of the other sketch postaggs have IDs here, the postagg IDs should really go in `PostAggregatorIds`